docs/onboarding_admins.rst

```diff
@@ -0,0 +1,97 @@
+Each *Admin Workstation* will have its own SSH keypair.
+
+This chapter assumes that you have one working *Admin Workstation*. If you've
+not completed that part of the setup yet, see :doc:`set_up_admin_tails`.  If
+your *Admin Workstation* is corrupted or lost, and you don't have a
+:doc:`backup <backup_workstations>`, see :doc:`rebuild_admin`.
+
+.. important::
+
+   If you make configuration changes on your servers using one
+   *Admin Workstation*, they may be overwritten by another *Admin Workstation*
+   if its local copy of the configuration is not identical. When working
+   with multiple admins, it is therefore important to establish protocols
+   for coordinating configuration changes. See :ref:`multiple_admins`.
+
+To onboard an additional administrator, you will need:
+
+- your existing *Admin Workstation* USB drive (referred to as **AW1** below)
+- an additional empty USB drive (referred to as **AW2** below)
+
+To set up AW2, follow these steps:
+
+1. Boot AW1, `unlock its persistent volume <https://tails.boum.org/doc/first_steps/persistence/use/index.en.html>`__,
+   and `set an admin password on the welcome screen <https://tails.boum.org/doc/first_steps/welcome_screen/administration_password/>`__
+2. Ensure that Tails and the SecureDrop version on AW1 are up-to-date.
+   If not, update now.
+3. Insert the empty AW2 USB drive.
+4. Launch the Tails installer (**Applications ▸ Tails ▸ Tails Installer**).
+5. Install Tails on AW2. This will delete all data on the AW2 USB drive.
+6. Shut down AW1.
+7. Boot AW2.
+8. Configure its persistent volume (**Applications ▸ Tails ▸ Configure persistent volume**).
+   Set a unique passphrase for AW2 and record it securely. Enable all persistence options.
+9. Reboot AW2, unlock its persistent volume, and set an admin password on the welcome screen.
+10. Open the file manager (**Applications ▸ Accessories ▸ Files**).
+11. Insert AW1. It should show up in the list of storage devices in the file manager under
+    a label like "7.0 GB Encrypted". Click the label and enter the drive
+    password when prompted to unlock it.
+12. In a terminal, type the following command:
+
+    ``rsync -a /media/amnesia/TailsData/Persistent/securedrop ~/Persistent``
+
+    This will copy *only* the ``securedrop`` directory from AW1 to AW2.
+13. Generate a new keypair on AW2 using the following command:
+
+    ``ssh-keygen -t rsa -b 4096``
+
+    When prompted, store the keypair in the default location.
+14. Shut down AW2.
+15. Boot AW1, unlock its persistent volume, and set an admin password on
+    the welcome screen.
+16. Open the file manager (**Applications ▸ Accessories ▸ Files**).
+17. Insert AW2 and unlock it.
+18. In a terminal, type the following commands to authorize the newly created SSH keypair
+    on your servers:
+
+    ``ssh-copy-id app -i /media/amnesia/TailsData/openssh-client/id_rsa.pub``
+    ``ssh-copy-id mon -i /media/amnesia/TailsData/openssh-client/id_rsa.pub``
```

Comment:
`ssh-copy-id -i /path/to/public/key user@host` works for me, but host then identity file gives the following error:

```
/usr/bin/ssh-copy-id: ERROR: Too many arguments.  Expecting a target hostname, got: 'app' '/home/user/.ssh/id_rsa_testtemp.pub'

Usage: /usr/bin/ssh-copy-id [-h|-?|-f|-n] [-i [identity_file]] [-p port] [[-o <ssh -o options>] ...] [user@]hostname
	-f: force mode -- copy keys without trying to check if they are already installed
	-n: dry run    -- no keys are actually copied
	-h|-?: print this help
```

Reply:
Good catch, thanks! We can omit the username here (it is defined in `~/.ssh/config`) but the arguments were indeed in the wrong order.